packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/decode/DecodeForm.tsx

```diff
@@ -22,36 +23,48 @@ import {
 import { Input } from "../../../../../../components/ui/Input";
 import { Skeleton } from "../../../../../../components/ui/Skeleton";
 import { cn } from "../../../../../../utils";
+import { useSystemAbisQuery } from "../../../../queries/useSystemAbisQuery";
 import { useWorldAbiQuery } from "../../../../queries/useWorldAbiQuery";
 import { getErrorSelector } from "./getErrorSelector";
 
 const formSchema = z.object({
   selector: z.string().min(1).optional(),
 });
 
+function isFunction(item: AbiItem): item is AbiFunction {
+  return item.type === "function";
+}
+
+function isError(item: AbiItem): item is AbiError {
```

Comment:
if you wanted to avoid the `AbiError` import, you could also narrow 
```suggestion
function isError(item: AbiItem): item is AbiItem & { type: "error" } {
```

Reply:
maybe worth clarifying the names of these `isAbiError` and `isAbiFunction`? since JS has errors/functions and is a bit confusing

Reply:
yeah makes sense, renamed to `isAbiError` and `isAbiFunction`. And also got rid of `AbiError` import and declared it locally, actually prefer that to the whole import, good suggestion